src/sections/header.liquid

```diff
@@ -103,12 +121,53 @@
       {% endfor %}
     </ul>
   </nav>
-</div>
+</section>
+
+<style>
+  {% if section.settings.logo != blank %}
```

Comment:
I just saw something, sorry 😞 Can you move the `if` statement above the `<style>` (just to avoid having a `<style>` with nothing inside when you don't have a logo)?

Reply:
@t-kelly beat me on it 😮

Reply:
Good catch!